devdata/env.json

```diff
@@ -1,4 +1,5 @@
 {
-  "RPA_WORKITEMS_ADAPTER": "RPA.Robocloud.Items.FileAdapter",
-  "RPA_WORKITEMS_PATH": "%{ROBOT_ROOT}/devdata/items1.json"
-}
+  "RPA_WORKITEMS_ADAPTER": "RPA.Robocorp.WorkItems.FileAdapter",
+  "RPA_INPUT_WORKITEM_PATH": "%{ROBOT_ROOT}/devdata/work-items-in/input/work-items.json",
+  "RPA_OUTPUT_WORKITEM_PATH": "%{ROBOT_ROOT}/devdata/work-items-out/run-1/work-items.json"
```

Comment:
Hopefully these are overridden by VSCode when playing with work items now. Last time we checked some time ago  they were interfering with extension's on-the-fly setting of them.